taskcluster/kinds/bicleaner-model/kind.yml

```diff
@@ -0,0 +1,73 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.task_context
+    - taskgraph.transforms.run:transforms
+    - translations_taskgraph.transforms.cached_tasks:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - toolchain
+
+tasks:
+    bicleaner-model:
+        label: bicleaner-model-{src_locale}-{trg_locale}
+        description: bicleaner-ai model for {src_locale}-{trg_locale}
+        attributes:
+            stage: bicleaner-model
+            src_locale: "{src_locale}"
+            trg_locale: "{trg_locale}"
+            cache:
+                type: bicleaner-model
+                resources:
+                    - pipeline/bicleaner/download_pack.py
+                    - pipeline/bicleaner/requirements/bicleaner-ai.txt
+        task-context:
+            from-parameters:
+                src_locale: training_config.experiment.src
+                trg_locale: training_config.experiment.trg
+            substitution-fields:
+                - description
+                - name
+                - label
+                - attributes
+                - run.command
+        worker-type: b-cpu
+        expires-after: "90 days"
```

Comment:
note: expiring this after 90 days will almost certainly cause everything downstream of them (like training task) to expire at the same time. Also, this expiry is only partly related to how long something stays cached -- the task expiring is one way a task will no longer be cached, but we have other ways to refresh a task in the cache (eg: by changing something that affects the cache digest).

I'm assuming that the reason you've set this to 90 days is to make sure we don't use outdated models for too long. If that is true, I suggest that we find a way to easily change the cache digest instead. That way we can cache models for as long as possible (eg: for a long lived training branch), and force a refresh through other means. The most straightforward way to do this would be to add a new file on disk to `resources`, which we could change anytime we want to refresh the bicleaner model cache.

Reply:
Interesting, I didn't know it works that way. I'll remove it.